ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml

```diff
@@ -17,22 +13,6 @@ resources:
       memory: 250Mi
       cpu: 100m
 
-images:
```

Comment:
Why are we removing this key?

Reply:
As of now, we don't use these images for any tests.